vector/src/main/java/im/vector/app/features/home/HomeActivity.kt

```diff
@@ -537,9 +537,7 @@ class HomeActivity :
         serverBackupStatusViewModel.refreshRemoteStateIfNeeded()
 
         // Check nightly
-        if (isFirstCreation()) {
-            nightlyProxy.updateIfNewReleaseAvailable()
-        }
+        nightlyProxy.mayDisplayFirebasePopup()
```

Comment:
maybe we could avoid mentioning `firebase` here and instead rely on the entry point, something like `NightlyProxy.onHomeResumed`, what do you think?

Reply:
Yes, it's better, thanks. I was wondering if we want to call this also on LoginActivity, but I guess this is not necessary.

e5773203010f88fa2c05297d0d1cfd51d9448e91